### PR TITLE
[SDK 2988] Add PE Variables 

### PR DESCRIPTION
**Pull request by shivamsharma2710**
- Supports PE related new APIs through plugin specific Android module.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 ## CHANGE LOG
 
+### Version 1.7.0 *(7th June 2023)*
```
Comment (darshanclevertap) [inline CHANGELOG.md:3]: @shivamsharma2710 change to today's date

File: example/android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -46,10 +46,14 @@
             android:value="2" />
         <meta-data
             android:name="CLEVERTAP_ACCOUNT_ID"
-            android:value="TEST-65R-487-R85Z"/>
+            android:value="8RK-KR9-976Z"/>
```
Comment (darshanclevertap) [inline example/android/app/src/main/AndroidManifest.xml:49]: @shivamsharma2710 remove credentials before merging

File: example/android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -46,10 +46,14 @@
...
         <meta-data
             android:name="CLEVERTAP_TOKEN"
-            android:value="TEST-ab4-256"/>
+            android:value="c23-32b"/>
```
Comment (darshanclevertap) [inline example/android/app/src/main/AndroidManifest.xml:52]: @akashvercetti Remove credentials before merging

File: example/ios/Runner/Info.plist [hunk 4]
```diff
@@ -21,9 +21,11 @@
 	<key>CFBundleVersion</key>
 	<string>$(FLUTTER_BUILD_NUMBER)</string>
 	<key>CleverTapAccountID</key>
-	<string>TEST-65R-487-R85Z</string>
+	<string>7Z8-W9R-876Z</string>
```
Comment (darshanclevertap) [inline example/ios/Runner/Info.plist:24]: @akashvercetti remove credentials before merging

File: example/android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (shivamsharma2710) [reply example/android/app/src/main/AndroidManifest.xml]: This is done.

File: CHANGELOG.md (hunk 1 above)
Comment (shivamsharma2710) [reply CHANGELOG.md]: This is done.

File: example/ios/Runner/Info.plist (hunk 4 above)
Comment (shivamsharma2710) [reply example/ios/Runner/Info.plist]: This is done.

**Pull request merged**